x-pack/plugins/security_solution/public/detections/components/host_isolation/use_external_host_isolation.tsx

```diff
@@ -22,9 +22,9 @@ interface ErrorType {
 }
 
 // TODO: 8.15: Remove `useGetSentinelOneAgentStatus` function when `agentStatusClientEnabled` is enabled/removed
```

Comment:
This comment should be updated or removed.

Reply:
@rylnd Thank you for the review and suggestions, you're absolutely right, I'll make the changes :+1: